apps/institution-service/cypress/e2e/aggregators.cy.ts

```diff
@@ -0,0 +1,49 @@
+import { Aggregator } from "models/aggregator";
+import { PORT } from "../../src/shared/const";
+import {
+  AGGREGATOR_USER_ACCESS_TOKEN_ENV,
+  SUPER_USER_ACCESS_TOKEN_ENV,
+  USER_ACCESS_TOKEN_ENV,
+} from "../shared/constants/accessTokens";
+import { createAuthorizationHeader } from "../shared/utils/authorization";
+import { runTokenInvalidCheck } from "../support/utils";
+
+const checkGetAggregatorsWithToken = (token: string) => {
+  cy.request({
+    url: `http://localhost:${PORT}/aggregators`,
+    method: "GET",
+    headers: {
+      Authorization: createAuthorizationHeader(token),
+    },
+  }).then((response: Cypress.Response<{ aggregators: Aggregator[] }>) => {
+    expect(response.status).to.eq(200);
+
+    expect(response.body.aggregators.length).to.be.greaterThan(0);
+    response.body.aggregators.forEach((aggregator) => {
+      ["id", "name", "displayName", "logo", "createdAt", "updatedAt"].forEach(
+        (attribute) => {
+          expect(aggregator).to.haveOwnProperty(attribute);
+        },
+      );
+    });
+  });
+};
+
+describe("/aggregators GET", () => {
+  it("returns a list of aggregators for a super admin", () => {
```

Comment:
Because there is no scope checking I'd just do 2 tests. 1 for valid token and 1 for invalid token. We don't have any logic that separates the roles on this endpoint.